src/js/views/patients/widgets/widgets.scss

```diff
@@ -7,14 +7,17 @@
 }
 
 .widgets__form-widget {
+  align-items: center;
+  display: flex;
+  max-width: 204px;
+  min-width: 0;
+  padding: 8px 8px 8px 16px;
```

Comment:
I think this should just be `padding-right: 8px;` leave the rest of the padding to the button